### PR TITLE
[fontbe] pass explicit axis_count when constructing Gvar

### PR DESCRIPTION
**Pull request by anthrotype**
Will fix https://github.com/googlefonts/fontc/issues/815

CI will be 🔴 until https://github.com/googlefonts/fontations/pull/919 is merged write-fonts dependency bumped

### CODE REVIEW

File: fontbe/src/gvar.rs [hunk 1]
```diff
@@ -52,6 +52,14 @@ impl Work<Context, AnyWorkId, Error> for GvarWork {
...
         let static_metadata = context.ir.static_metadata.get();
         let axis_order: Vec<_> = static_metadata.axes.iter().map(|a| a.tag).collect();
+        let axis_count: u16 = axis_order
+            .len()
+            .try_into()
+            // in the unlikely event that we have more than 65535 axes...
+            .map_err(|_| Error::OutOfBounds {
+                what: "axis count".into(),
+                value: axis_order.len().to_string(),
+            })?;
```
Comment (cmyr) [inline fontbe/src/gvar.rs:62]: I would personally unwrap here, since I can't imagine this being possible for any conceivable font, but 🤷

**Review by cmyr — APPROVED** [review]
looks good!

File: fontbe/src/gvar.rs (hunk 1 above)
Comment (anthrotype) [reply fontbe/src/gvar.rs:62]: just wait a few years :)

Comment (rsheeter) [reply fontbe/src/gvar.rs:62]: I strongly prefer Result unless it _can't_ happen. Rationale: if we unwrap all the things that shouldn't happen but plausibly can then it's all but assured we'll hit _some_ of them in production and rue the day we didn't use Result.

**Pull request merged**